packages/next-urql/examples/1-with-urql-client/.gitignore

```diff
@@ -0,0 +1,25 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
We might want to add these to the global gitignore instead to reduce files (same for the logo maybe use them from `/docs`)

Reply:
I think it’s fine to make an exception for examples until those are cleaned up maybe? 🤔

Reply:
✅ Agreed, will do.